web/src/main/resources/assets/js/map.js

```diff
@@ -285,23 +285,24 @@ module.exports.addElevation = function (geoJsonFeature, useMiles, details) {
         GHFeatureCollection.push(elevationCollection);
     }
 
-
     if (elevationControl === null) {
         // TODO no option to switch to miles yet
-        // TODO think about adding Color Mappings
-        elevationControl = L.control.heightgraph({
-             width: 800,
-             height: 280,
-             margins: {
-                 top: 10,
-                 right: 30,
-                 bottom: 55,
-                 left: 50
-             },
-             position: "bottomright"
-         });
-
-         elevationControl.addTo(map);
+        options = {
+           width: 800,
+           height: 280,
+           margins: {
+               top: 10,
+               right: 30,
+               bottom: 55,
+               left: 50
+           },
+           position: "bottomright"
+        }
+        // for path details use random colors
+        if(jQuery.isEmptyObject(details))
```

Comment:
Here jquery is required as details is `{}` if no details parameter was specified

Reply:
We could add this part [here](https://github.com/graphhopper/graphhopper/pull/1569/files#diff-e1a9691b5e85c253cc337b86010c30abR274) and avoid using jQuery.